docs/UserGuide.adoc

```diff
@@ -151,18 +151,18 @@ Provides statistics about the pet tracker and schedule system, such as total num
 
 Adds a pet to the pet tracker system.
 
-Format: `addpet n/NAME a/AGE s/SPECIES b/BREED g/GENDER f/FOOD [t/TAG]...`
+Format: `addpet n/NAME g/GENDER b/DATE OF BIRTH s/SPECIES f/FOOD : AMOUNT [t/TAG]...`
```

Comment:
is the space before and after : intended? Our current implementation is `food:amount` not `food : amount` right?

Reply:
actually it's the same, cuz the spaces will be trimmed